build(docs-infra): update in-memory-we-api and karma-jasmine-html-reporter version

santoshyadavdev opened the pull request:
Fixes #29802

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: #29802


## What is the new behavior?


## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

mary-poppins commented:
You can preview 389e173 at https://pr32892-389e173.ngbuilds.io/.

gkalpak reviewed (approved):
Thx, @santoshyadav198613 :+1:
Can you please update the commit message to `build(docs-infra): ...`.

Also, (although not directly related to this PR) I noticed that `npm test` on examples fails due to [this issue with `karma-jasmine-html-reporter`][1]. If you could update to the latest `karma-jasmine-html-reporter` too (in this PR or another one), that would be rad (but is not blocking this PR) 🔥

[1]: https://github.com/taras42/karma-jasmine-html-reporter/issues/31

santoshyadav198613 commented:
> Thx, @santoshyadav198613 
> Can you please update the commit message to `build(docs-infra): ...`.
> 
> Also, (although not directly related to this PR) I noticed that `npm test` on examples fails due to [this issue with `karma-jasmine-html-reporter`](https://github.com/taras42/karma-jasmine-html-reporter/issues/31). If you could update to the latest `karma-jasmine-html-reporter` too (in this PR or another one), that would be rad (but is not blocking this PR)

Will do that captain :)

santoshyadav198613 commented:
Done with the changes.

mary-poppins commented:
You can preview 7f642a1 at https://pr32892-7f642a1.ngbuilds.io/.

gkalpak commented on aio/tools/examples/shared/package.json:
```diff
@@ -69,7 +69,7 @@
     "karma-coverage-istanbul-reporter": "~2.0.1",
     "karma-jasmine": "~2.0.1",
-    "karma-jasmine-html-reporter": "^0.2.2",
+    "karma-jasmine-html-reporter": "^1.4.2",
```
Thx for making the change. Can you, please, also update the commit message (to indicate that this is not just about `in-memory-web-api` (or move this change to a separate PR) and also mention that this fixes taras42/karma-jasmine-html-reporter#31?

gkalpak replied:
Thx, can you please mention that the `karma-jasmine-html-reporter` fixes lso mention that this fixes taras42/karma-jasmine-html-reporter#31 in the commit message?

santoshyadav198613 replied:
Let me know if its fine I was not able to mention that in the message so added in Fixes

mary-poppins commented:
You can preview 4281b00 at https://pr32892-4281b00.ngbuilds.io/.

mary-poppins commented:
You can preview c28cec2 at https://pr32892-c28cec2.ngbuilds.io/.

santoshyadav198613 commented:
Hi @gkalpak ,
Let me know if any changes are needed,

gkalpak commented:
Thx for the ping, @santoshyadav198613. I updated the commit message to be more accurate and marked this for merging.

mary-poppins commented:
You can preview 9ac5353 at https://pr32892-9ac5353.ngbuilds.io/.

santoshyadav198613 commented:
Hi @gkalpak ,
Ci failed with  error Command "build-with-ivy-ci" not found. Should i sync the branch?

gkalpak commented:
Yes, please ❤️

mary-poppins commented:
You can preview 6da1718 at https://pr32892-6da1718.ngbuilds.io/.

santoshyadavdev committed:
build(docs-infra): update in-memory-we-api and karma-jasmine-html-reporter version

The `karma-jasmine-html-reporter` update also includes a fix for
taras42/karma-jasmine-html-reporter#31.

Fixes #29802

mary-poppins commented:
You can preview dc510ba at https://pr32892-dc510ba.ngbuilds.io/.

santoshyadav198613 commented:
> Yes, please

Hi @gkalpak ,
Done, and CI is green now.
